pkg/diagnostics/service_monitoring.go

```diff
@@ -23,13 +23,15 @@ var (
 	namespaceKey        = tag.MustNewKey("namespace")
 	resiliencyNameKey   = tag.MustNewKey("name")
 	policyKey           = tag.MustNewKey("policy")
+	errorCodeKey        = tag.MustNewKey("errorcode")
```

Comment:
nit: follow the same pattern we have on other keys error_code or errorCode (we have both, choose one I guess)